config_models/models.py

```diff
@@ -142,35 +142,61 @@ def cache_key_name(cls, *args):
             return 'configuration/{}/current'.format(cls.__name__)
 
     @classmethod
-    def current(cls, *args):
+    def current(cls, *args, **kwargs):
         """
         Return the active configuration entry, either from cache,
         from the database, or by creating a new empty entry (which is not
         persisted).
+
+        Keyword Arguments:
+            query_db: if False, an empty entry will always be returned in
```

Comment:
I worry about this. Using the default value is likely not correct behavior. I'd rather an explicit exception here if the value isn't in cache and we want to prevent the db query.